packages/react/src/components/cookieConsent/modal/Modal.tsx

```diff
@@ -19,32 +19,63 @@ export function Modal(): React.ReactElement | null {
   // if hasUserHandledAllConsents() was false at first and then later true, user must have saved consents.
   const showScreenReaderSaveNotification = isModalInitiallyShown && !shouldShowModal;
   const { settingsSaved } = useCookieConsentUiTexts();
-  const containerId = 'HdsCookieConsentContainer';
+  const containerId = 'HdsCookieConsentModalContainer';
+  const placeholderId = 'HdsCookieConsentModalPlaceholder';
   const getContainerElement = (): HTMLElement | null => document.getElementById(containerId);
+  const getPlaceholderElement = (): HTMLElement | null => document.getElementById(placeholderId);
   const [isDomReady, setIsDomReady] = useState<boolean>(false);
+  const [contentHeight, setContentHeight] = useState<number>(0);
+  const modalContentRef = useRef<HTMLDivElement>();
+  const bodyBottomPaddingStyleRef = React.useRef<string>(null);
```

Comment:
We do not seem to add this ref to any react node, or update its value? Do we need it?